uint128.go

```diff
@@ -22,9 +22,11 @@ func powerOfTen128(n int) uint128T {
 	if n < 0 {
 		n = -n
 	}
-	b := n % 19
-	a := n - b
-	return umul64(powersOf10[a], powersOf10[b])
+
+	if n > 19 {
```

Comment:
I can see we come to this section when we run through the matchScales128() function. Why is the range for matching both exponents 19?

Reply:
10 **19 is the greatest uint64
Anything above that needs to be expanded into uint128T